integration/run_test.go

```diff
@@ -46,22 +45,22 @@ import (
 	"testing"
 	"time"
 
+	"github.com/awslabs/soci-snapshotter/config"
 	"github.com/awslabs/soci-snapshotter/soci/store"
 	shell "github.com/awslabs/soci-snapshotter/util/dockershell"
 	"github.com/awslabs/soci-snapshotter/util/testutil"
 )
 
 func TestRunWithDefaultConfig(t *testing.T) {
-	b, err := os.ReadFile("../config/config.toml") // example toml file
+	cfg, err := config.NewConfigFromToml("../config/config.toml") // example toml file
```

Comment:
Any reason we don't just read the file and use it as the snapshotter TOML? If we read the file and output it to a string we can just not use `getSnapshotterConfigToml`.

nit: While you're here I wonder if we could at least drag the path out as some sort of const

Reply:
Yeah, we can do that. That should fix the weird issue where we had to comment out `disable_verification` in the default config too.